### PR TITLE
Migrate Beefy 101-javaee-like-getting-started

### LINKED ISSUE #90: Migrate 101-javaee-like-getting-started from the Quarkus Beefy Test Suite

### PR DESCRIPTION
**Pull request by pjgg**
javaee-like-getting-started migration

Migrate101-javaee-like-getting-started from Beefy-ts to quarkus-test-suite

FIX #90

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -135,6 +135,25 @@ It also verifies multiple deployment strategies like:
 ### `http-advanced`
 Verifies Server/Client http_2/1.1, Grpc and http redirections. 
 
+
+# scenario-101-getting-started
```
Comment (Sgitario) [inline README.md:139]:
```suggestion
### `javaee-like-getting-started`
```

**Review by Sgitario — CHANGES_REQUESTED** [review]
Just a minor thing in the README.md.
I don't see any scenarios for OpenShift, don't you see any value in adding some? (just asking)

File: README.md (hunk 1 above)
Comment (pjgg) [reply README.md]: I don't see it either

File: BEEFY_TS_MIGRATION_STATUS.md [hunk 2]
```diff
@@ -7,5 +7,7 @@
 | 020-quarkus-http-non-application-endpoints | http/http-advanced |
 | 021-quarkus-panache-multiple-pus | sql-db/multiple-pus |
 | 023-quarkus-kamelet | kamelet |
+| 101-javaee-like-getting-started | javaee-like-getting-started |
 | 302-quarkus-vertx-jwt | security/vertx-jwt|
 | 304-quarkus-vertx-routes | http/reactive-routes |
+| 302-quarkus-vertx-jwt | security/vertx-jwt|
```
Comment (Sgitario) [inline BEEFY_TS_MIGRATION_STATUS.md:13]: This line is duplicated

**Review by Sgitario — CHANGES_REQUESTED** [review]

**Review by Sgitario — APPROVED** [review]

**Pull request merged**